kind/kind.sh

```diff
@@ -156,6 +156,7 @@ show_usage() {
     log_message "0" ""
     log_message "0" "\t--cluster-name <name>\t\t\tName of the cluster. Default: kind"
     log_message "0" "\t--delete-kind-cluster\t\t\tDeletes the Kind cluster prior to creating a new one. Default: No"
+    log_message "0" "\t--deploy-sample-application\t\tDeploys a sample application based on nginx. Default: No"
```

Comment:
Should we do that ? Maintaining examples up to date with what we package/install is not so easy !

Reply:
I see the problem. I'll remove the sample then.

Reply:
Removed the sample application deployment from the kind installation process.